mysite/local_settings.py

```diff
@@ -0,0 +1,12 @@
+import os
```

Comment:
This file is committed by mistake. `git rm --cached mysite/local_settings.py; echo 'local_settings.py' >> .gitignore; git add .gitignore` would fix this.